terraform/modules/infra-ecs-fargate/main.tf

```diff
@@ -0,0 +1,248 @@
+locals {
+    canaries_security_group = "sg-044ef7bc34691164a"
```

Comment:
is there any reason why the security group is in locals and not in var?

Reply:
Let's parametrize it! Added in the following commit. Thanks